src/shared/Core/Authentication/MicrosoftAuthentication.cs

```diff
@@ -478,7 +478,7 @@ public static bool CanUseBroker(ICommandContext context)
 
             return defaultValue;
 #else
-            // OS broker requires .NET Framework right now until we migrate to .NET 5.0 (net6.0-windows10.x.y.z)
+            // OS broker requires .NET Framework right now until we migrate to .NET 5.0 (net5.0-windows10.x.y.z)
```

Comment:
What is the reason to _not_ update the comment?

Reply:
I don't really have good reasoning, when I noticed the change, I just kind of thought that it would be a good idea to leave that comment alone. I can revert that if you'd like.

Reply:
It's more of a dig to clean up this `#if`, so I'm happy to leave it. (Do we still have some need to support compiling on .NET Framework?)

The change makes the comment more consistent, so I'd say feel free to leave it as you have it.

Reply:
Yes, we have to compile on .NET Framework on Windows (see the note in [this section](https://github.com/GitCredentialManager/git-credential-manager/blob/main/docs/architecture.md#overview) of `Architecture.md`).